### PR TITLE
Bump msbuild to track mono-2018-04

### PR DESCRIPTION
**Pull request by radical**
.. to pick up a fix for the linux build.

### CODE REVIEW

**Comment by akoeplinger** [comment]
@monojenkins build OS X Extended

**Comment by radical** [comment]
@monojenkins build OS X Extended

**Comment by radical** [comment]
@akoeplinger should be fixed now

File: scripts/ci/run-test-mac-sdk.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ export PATH=${MONO_REPO_ROOT}/external/bockbuild/stage/bin:$PATH
 
 # Bundled MSBuild
 cd ${MONO_REPO_ROOT}/external/bockbuild/builds/msbuild-15/
-${TESTCMD} --label="msbuild-tests" --timeout=180m ./build.sh -host mono -configuration Release
+${TESTCMD} --label="msbuild-tests" --timeout=180m ./build.sh -hostType mono -configuration Release
 zip ${MONO_REPO_ROOT}/msbuild-test-results.zip artifacts/2/Release-MONO/TestResults/*
```
Comment (akoeplinger) [inline scripts/ci/run-test-mac-sdk.sh]: I guess this needs updating too:

```
04:28:23 zip error: Nothing to do! (/Users/builder/jenkins/workspace/test-mono-pull-request-mac-sdk/msbuild-test-results.zip)
```

**Pull request closed without merging**